Gemfile

```diff
@@ -2,6 +2,7 @@
 
 source 'https://rubygems.org'
 
+gem 'aws-sdk-core'
```

Comment:
Is it possible to lock down the version here? Since it's a new gem anyway, we shouldn't expect backward incompat issue. If there's any correlation to aws-sdk version related to inspec, please add .a todo note.